src/lib/core/components/VariableLink.tsx

```diff
@@ -1,9 +1,13 @@
 /*
  * Link to the page for a variable
  */
-import { forwardRef, Ref } from 'react';
+import { CSSProperties, forwardRef, Ref } from 'react';
 import { Link, LinkProps } from 'react-router-dom';
 
+const disabledStyle: React.CSSProperties = {
+  pointerEvents: 'none',
+};
+
```

Comment:
This can be removed since A) `pointerEvents` would disallow the cursor styles and B) it's being overridden by the component-scoped `disabledStyle` anyway.

Reply:
Oops, thanks!